tests/ignite/contrib/metrics/test_cohen_kappa.py

```diff
@@ -0,0 +1,208 @@
+import os
+
+import numpy as np
+import pytest
+import torch
+from sklearn.metrics import cohen_kappa_score
+
+import ignite.distributed as idist
+from ignite.contrib.metrics import CohenKappa
+from ignite.engine import Engine
+from ignite.exceptions import NotComputableError
+
+
+def test_no_update():
+    ck = CohenKappa()
+
+    with pytest.raises(NotComputableError):
+        ck.compute()
+
+
+def test_input_types():
+    ck = CohenKappa()
+
+    with pytest.raises(ValueError):
```

Comment:
Please, use `with pytest.raises(ValueError, match=r"...")` to match exact error. You can match first 2-3 significant words. Same for all `with pytest.raises(` below. Above `with pytest.raises(NotComputableError)` is OK.

Reply:
That's done too. Check the recent commit.

Reply:
@KickItLikeShika, sorry, I do not understand why did you remove all the tests like 
```python
    with pytest.raises(ValueError):
        CohenKappa(weights="ss")
```
I think I asked you to remove redundent ones only and not all, right. Am I missing something ?

Reply:
Anyway, thanks for the update, let's restore removed tests **only relevant ones** and it will be good to go.

Reply:
I thought you told me to remove all of them! Sorry i will add them back